README.md

```diff
@@ -88,15 +88,19 @@ If the build is not for a PR or the master branch an error will be thrown with t
 
 ### getVariables
 
-Takes the repository name as a parameter, i.e. `ffc-demo-web`, and returns information required by the build steps as an array
+Takes the repository name as a parameter, i.e. `ffc-demo-web`, as well as the name of the relevant function to call depending on the project type.
```

Comment:
I'd say that it takes a version parameter as opposed to a function.  As supplying '1.0.0' would work just as well with this setup.  We just have a helper method to work out the version number for us and pass it in.

Reply:
Amended as suggested.